XCTest.xcodeproj/project.pbxproj

```diff
@@ -409,13 +407,22 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				INFOPLIST_FILE = Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				MACOSX_DEPLOYMENT_TARGET = 10.11;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
-				SDKROOT = macosx;
```

Comment:
In my testing, Xcode will automatically adjust the `SDKROOT` used for a build such that it matches the platform of the run destination. For example, if the selected SDK is "Latest macOS" and the build has an iOS device destination selected, it will implicitly use the "Latest iOS" SDK instead. Therefore I would prefer keeping the Base SDK setting at the project level with no target overrides.

One other minor point: I think it would be better to specify the new `SUPPORTED_PLATFORMS` value at the level of the `SwiftXCTest` target, rather than at the project level, for now anyway. This is because the `SwiftXCTestFunctionalTests` target only works on macOS, and I don't foresee that changing.